packages/react/src/CTABanner/CTABanner.visual.spec.ts

```diff
@@ -43,6 +43,15 @@ test.describe('Visual Comparison: CTABanner', () => {
     expect(await page.screenshot()).toMatchSnapshot()
   })
 
+  test('CTABanner / With Trailing Component', async ({page}) => {
```

Comment:
Can't see the new snapshot image in this PR. Could you try running it again please?

Reply:
Good spot 👍 Looks like they've generated now, although I'm not sure why this has changed

`packages/react/src/Tooltip/Tooltip.visual.spec.ts-snapshots/Visual-Comparison-Tooltip-Tooltip-Label-Type-1-linux.png`

Reply:
Ah nvm, looks like they just weren't regenerated after this commit 

https://github.com/primer/brand/pull/547/commits/015bfea49cd33fd3065dfc4c7e7b742b2a45647d#diff-7e60c94a63a2596dd7d646b8d1bbd5bc87baf66db760adc47f70150c93b842e5

Looks fine to me